target/agon/include/romdisk_info_h.asm

```diff
@@ -1,4 +1,4 @@
 IFNDEF ROMDISK_H
```

Comment:
This file is auto-generated at build time, it would be better to remove it from the repo